### PR TITLE
Mm 44564 - fix Drop zone UI alignment is off-center 

### PR DESCRIPTION
**Pull request by hokandil**
this pull request addresses a Help Wanted ticket:

  Fixes https://github.com/mattermost/mattermost-server/issues/21395

 JIRA ticket.  https://mattermost.atlassian.net/browse/MM-44564

MM-44564

**Release Note**
```release-note
NONE
```

### CODE REVIEW

**Review by hokandil — COMMENTED** [review]
lint fix

**Review by hokandil — COMMENTED** [review]
Lint fixing

**Review by hokandil — COMMENTED** [review]
Lint fixes

**Review by hokandil — COMMENTED** [review]
Indentation lint fix

File: sass/responsive/_tablet.scss [hunk 1]
```diff
@@ -147,6 +147,10 @@
             #channel-header {
                 width: calc(100% - 400px);
             }
+            //adjust file drop zone
+            #file-overlay {
```
Comment (M-ZubairAhmed) [inline sass/responsive/_tablet.scss:151]: nit : `file-upload-overlay` 
just for clarity

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
minor nit

**Comment by M-ZubairAhmed** [comment]
@hokandil let us know if you face problems fixing ci failure :)

**Review by hokandil — COMMENTED** [review]
nit : file-upload-overlay
just for clarity

File: sass/responsive/_tablet.scss [hunk 2]
```diff
@@ -147,6 +147,10 @@
             #channel-header {
                 width: calc(100% - 400px);
             }
+            //adjust file drop zone
+            #file-upload-overlay {
+                width: calc(100% - 400px);
```
Comment (BenCookie95) [inline sass/responsive/_tablet.scss:152]: What would happen if you expand the RHS view? That increases the RHS width to 960px
Screen Shot 2022-10-26 at 9 11 04 AM

**Review by BenCookie95 — CHANGES_REQUESTED** [review]

File: sass/responsive/_tablet.scss (hunk 2 above)
Comment (hokandil) [reply sass/responsive/_tablet.scss:152]: @BenCookie95 I just followed the channel header. I don't knew if I need to consider the RHS view expansion.

Comment (hokandil) [reply sass/responsive/_tablet.scss:152]: @BenCookie95 I did a test , and the drag and drop option is not available whenever the RHS is expanded.

Comment (BenCookie95) [reply sass/responsive/_tablet.scss:152]: I was able to test on the cloud test server. I couldn't see any issues, nice work

**Review by BenCookie95 — APPROVED** [review]
LGTM!

**Comment by M-ZubairAhmed** [comment]
@hokandil can you please resolve the conflicts thank you

**Review by matthewbirtch — APPROVED** [review]
Looks good @hokandil. Thanks for fixing this!

**Review by M-ZubairAhmed — APPROVED** [review]
💯 It looks good to me. Thank you 🎉

**Comment by M-ZubairAhmed** [comment]
@hokandil can you please fix the conflicts, so we can add QA review for this?

**Comment by hokandil** [comment]
Hi everyone,  I can see that the issue was resolved by  another PR , although it is not mentioned on that PR.
Let me close this PR , thank you.

**Comment by M-ZubairAhmed** [comment]
@hokandil which pr exactly?

**Pull request closed without merging**